spec/xcpretty/term_spec.rb

```diff
@@ -4,22 +4,22 @@ module XCPretty
   describe Term do
     it "marks unicode as unavailable" do
       Encoding.stub(:default_external).and_return(Encoding::ASCII)
-      Term.unicode?.should be_false
+      Term.unicode?.should be_falsey
```

Comment:
@chipp sure, they can be fixed by replacing `be_false` with either `be false` or `be_falsey`.

I already did it in `master` yesterday and merged your commit.
Thanks!